course02/homework02/Tomasz/short_circular_buffer.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <array>
+#include <cassert>
+
+template<typename _Ty, std::size_t _Size>
+class short_circular_buffer
```

Comment:
Nice, but it already exists, see boost.circular buffer

Reply:
Next time will definitely use that one! Thanks.

Reply:
Or actually it's probably time to learn boost. I will change that.